package.json

```diff
@@ -4361,6 +4361,7 @@
         "bytes": "^3.1.2",
         "cross-fetch": "^4.0.0",
         "cross-spawn": "^7.0.3",
+        "crypto-browserify": "^3.12.0",
```

Comment:
I thought the point of `src/browser/package.json` was to avoid needing this in both places (or vice versa)?

Reply:
Good find, I must have added this here accidentally. Updated to just src/browser/package.json